shellpython/shellpy.py

```diff
@@ -3,20 +3,21 @@
 import os
 import subprocess
 from shellpython.preprocessor import preprocess_file
+from argparse import ArgumentParser
+import re
 
 
 def main():
-    if len(sys.argv) == 1 or sys.argv[1] in ['-h', '--help', 'help']:
-        print('Shellpy, write shell scripts in Python easily')
-        print('Usage: shellpy [*.spy script] [arguments]')
-        exit(0)
+    def shellpyFile(fil):
```

Comment:
that is possible. shellpy args should come before name of script : shellpy [shellpy_args] script [script_args]. There is no other way to distinguish between the two.